generated/nitclk/nitclk_service.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <sstream>
 #include <fstream>
+#include <numeric>
```

Comment:
Why are we including numeric now?

Reply:
`std::accumulate` is in `numeric`.

Reply:
I'm trying to write a helper so it only gets included if we have a two-dimension size mechanism rn

Reply:
I've added some helper functions and we now only include numeric if we have a two dimensional array and need it.